frontend/components/EditorImagesImport/SvgTab.js

```diff
@@ -1,28 +1,59 @@
 import MaterialIcon from '@material/react-material-icon';
-import {Button } from '@material/react-typography';
+import { Button } from '@material/react-typography';
+import axios from 'axios';
+const apiUrl = process.env.API_URL || 'http://localhost:1337';
+import Cookies from "js-cookie";
 
 class SvgTab extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      imageUrl: "",
+    };
+    this.loadFile = this.loadFile.bind(this)
+  }
 
-	loadFile(event) {
-		console.log("run loadfile");
-	  var output = document.getElementById('output-image');
-	  output.src = URL.createObjectURL(event.target.files[0]);
-	};
-	
-	render() {
-		return(
-		  <div className="images-tab">
-		    <input type="file" name="images-tab-upload" id="images-tab-upload" accept="image/*" onChange={this.loadFile}/>
-		    <label htmlFor="images-tab-upload" className="images-tab-upload-label">
-		    	<div className="circle">
-		    		<MaterialIcon className="paperclip-icon" icon='attach_file' />
-		    		<Button>Upload your SVG here</Button>
-		    	</div>
-		    	<img id="output-image"/>
-		    </label>
-		  </div>
-		);
-	}
+  loadFile(e) {
+    e.preventDefault();
+
+    const formData = new FormData();
+    formData.append("files", e.target.files[0]);
+
+    axios
+      .post(apiUrl + '/upload', formData, {
+        headers: {
+          'Authorization': 'Bearer ' + Cookies.get("jwt")
+        }
+      })
+      .then(response => {
+        if (response.data[0].url.includes("storage.googleapis.com")) {
```

Comment:
If the URL returned from the server doesn't include a google storage domain, then need to prepend apiURL (because strapi only gives back `/route-to-file`)